_data/changelogs/template-documentation.yml

```
@@ -0,0 +1,16 @@
+title: Documentation page
+type: template
+items:
+  - date: NNNN-NN-NN
+    summary: Visual order of the Documentation page matches the HTML order.
+    summaryAdditional:  Now, there are two separate side navigation components for mobile and desktop that are displayed or hidden depending on viewport widths.
+    isBreaking: false
+    affectsAccessibility: true
+    affectsAssets:
+    affectsGuidance:
+    affectsJavascript:
+    affectsMarkup: false
```

I vote for marking this as a markup change. 

```suggestion
    affectsMarkup: true
```